new/LIT14518QeneOzyan.xml

```diff
@@ -0,0 +1,83 @@
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne with an allusion to the fall of Uzziah as described in <ref type="work" corresp="LIT3500Chroni">2 Chr. 26</ref> and 
+                    other biblical books. The type and author are not given in the manuscript. The verses end in ላ፡ and are marked with ።, ending up in 
+                    a series of possibly eight verses, so that the poem can be attributed the mawaddǝs type of qǝne poetry according to the 
+                    classification in <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>  
+                    (disregarding ተድላ፡ and ገሊላ፡ in <ref target="#l1"/> and <ref target="#l2"/>, which do not have verse markers and are therefore 
+                    not classified as verse ends in this record, but including a verse marker in <ref target="l5"/>, which is not 
+                    visible in the available pictures because of the tight binding of the manuscript and which was supplied by the editor). </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-11-01">Created entity.</change>
+            <change who="CH" when="2024-11-01">Updated edition with several readings supplied by Nafisa Valieva.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vc. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። .</note>
+                <ab>
+                    <l n="1">ዖዝያን፡ ወድቀ፡ እመንበረ፡ ተድላ፡ አመ፡ ፈቀደ፡ ይፀንዓን፡ ካህነተ፡ አሮን፡ ሰረገላ።</l> 
+                    <l n="2">እንዘ፡ ዲበ፡ ፈረስ፡ ቅይሕ፡ መንግሥተ፡ ገሊላ፡ ወአመ፡ ፈቀደ፡ 
+                        በ<del rend="strikethrough"><unclear>ቅ</unclear>ንሣ<unclear>አ፡</unclear></del>ዝንጋዔ፡ ያስተጋብእ፡ ፍሬ፡ ምህርካሁ፡ ለአማሌቅ፡ 
+                        ሰግላ።</l>
```

Comment:
እንዘ፡ ዲበ፡ ፈረስ፡ ቅይሕ፡ መንግሥተ፡ ገሊላ። - I would make it to a separate line, assuming four dot asterisc which is not visible due to the tight binding. Then you have 9 lines, right? mawaddǝs kwellekemu type.